src/pages/TransactionDetails/components/TransactionInfo/TransactionInfo.tsx

```diff
@@ -321,6 +324,26 @@ export const TransactionInfo = ({
                 </div>
               </DetailItem>
 
+              {receiver !== transaction.receiver && (
+                <DetailItem title='Destination'>
+                  <div className='d-flex flex-column'>
+                    <div className='d-flex align-items-center'>
+                      {isContract(receiver) ? (
+                        <span className='me-2 text-neutral-400'>Contract</span>
+                      ) : (
+                        ''
```

Comment:
Why return and empty string? Can the above condition not use `&&` instead of ternary?

Reply:
updated

Reply:
Great. 🔥